python/device/device.py

```diff
@@ -223,6 +265,7 @@ def get_longhaul_metrics(self):
 
         props = {
             "runStartUtc": self.metrics.run_start_utc.isoformat(),
+            "runCurrentTimeUtc": datetime.datetime.utcnow().isoformat(),
```

Comment:
start time of the run? latest update time?

Reply:
renaming